playbooks/appsemblerPlaybooks/dogwood_multiserver_deploy_cloud_sql.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+# Appsembler pro tier deployment
+# Docs: https://github.com/appsembler/openedx-docs/blob/master/openedx/pro_tier_deployment.md
+
+- name: Configure mongo
+  hosts: "mongo-server"
+  sudo: True
+  gather_facts: True
+  roles:
+    - { role: swapfile, SWAPFILE_SIZE: "2GB" }
+    - mongo
+    - oraclejdk
+    - elasticsearch
+    - memcached
+
+- name: Configure stateless edxapp server
+  hosts: "edxapp-server"
+  sudo: True
+  gather_facts: True
+  vars:
+    migrate_db: "yes"
+    openid_workaround: True
+    COMMON_ENABLE_DATADOG: False
+  roles:
+    - { role: swapfile, SWAPFILE_SIZE: "2GB" }
+    - mysql_init
+    - role: nginx
+      nginx_sites:
+      - certs
+      - cms
+      - lms
+      - forum
+      nginx_default_sites:
+      - lms
+    - { role: 'edxapp', celery_worker: True }
+    - edxapp
+    - analytics_api
+    - edx_notes_api
+    - demo
+    - oauth_client_setup
+    - forum
+    - certs
+    - edx_ansible
+    # scorm if desired.  requires more setup in server-vars
+    # see https://github.com/appsembler/ssla-scorm-player/blob/nyif/master/README.md
+    # - scorm
+
+- name: Install xqueue, notifier and rabbitmq
+  hosts: "mongo-server"
+  sudo: True
+  gather_facts: True
+  vars:
+    migrate_db: "yes"
+  roles:
+    - mysql_init
```

Comment:
@mrgnr If the role is not present in the roles list, xqueue fails at the migrate db task, because it cannot find the mysql vars for connection.